plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/ItemMetaConverterHack.java

```diff
@@ -51,7 +52,7 @@ public static void processItem(final ItemStack item) {
 			for (int i = 0, l = currentLore.size(); i < l; i++) {
 				final var loreLine = currentLore.get(i);
 				if (!ChatUtils.isBaseComponent(loreLine)) {
-					currentLore.set(i, Component.text().append(loreLine).build());
+					currentLore.set(i, Component.empty().append(loreLine.compact()));
```

Comment:
why not loreLine.compact() ?

Reply:
It's what worked. I'm just going to close this for now because the item issue didn't occur in 1.20.6.